pkgs/tools/wayland/chayang/default.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, stdenv
+, fetchFromSourcehut
+, meson
+, ninja
+, pkg-config
+, wayland-scanner
+, wayland-protocols
+, wayland
+}:
+
+stdenv.mkDerivation rec {
+  pname = "chayang";
+  version = "0.1.0";
+
+  src = fetchFromSourcehut {
+    owner = "~emersion";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-3Vu9/Bu2WQe2Yx/2BK25pEpuPNwX6g3qoFUMznCFHeI=";
+  };
+
+  strictDeps = true;
+
```

Ah sorry, seems like the doubled `pkg-config` was indeed needed for working cross. It looks for two different `pkg-config`s - once for `buildPlatform` to find a program from `nativeBuildInputs`, once for `hostPlatform` to find the libraries.

```suggestion

  pkgsBuildBuild = [
    pkg-config
  ];

```